app/tests/test_record_page.py

```diff
@@ -265,3 +265,83 @@ def test_returns_record_page_for_superuser(client, mock_superuser):
     response = client.get(f"/record/{file.FileId}")
 
     assert response.status_code == 200
+
+
+def test_record_search_box(client, mock_superuser):
+    mock_superuser(client)
+
+    file = FileFactory(
+        FileName="test_file.txt",
+        FilePath="data/content/folder_a/test_file.txt",
+        FileType="file",
+    )
+
+    metadata = {
+        "date_last_modified": "2023-02-25T10:12:47",
+        "closure_type": "Closed",
+        "description": "Test description",
+        "held_by": "Test holder",
+        "legal_status": "Test legal status",
+        "rights_copyright": "Test copyright",
+        "language": "English",
+    }
+
+    [
+        FileMetadataFactory(
+            file=file,
+            PropertyName=property_name,
+            Value=value,
+        )
+        for property_name, value in metadata.items()
+    ]
+
+    response = client.get(f"/record/{file.FileId}")
+
+    assert response.status_code == 200
+
+    html = response.data.decode()
+
+    search_html = """<p class="govuk-body search__heading">Search for digital records</p>"""
```

Comment:
you could consolidate and enhance these many frgamented assertions by just asserting the full html search component is in the html

e.g.

```
search_container_html = <div class="search__container govuk-grid-column-full">
    <div class="search__container__content">
        <p class="govuk-body search__heading">Search for digital records</p>
        <form method="post" action="/search">
            {{ form.csrf_token }}
            <div class="govuk-form-group govuk-form-group__search-form">
                <label for="searchInput"></label>
                <input class="govuk-input govuk-!-width-three-quarters"
                       id="searchInput"
                       name="query"
                       type="text">
                <button class="govuk-button govuk-button__search-button"
                        data-module="govuk-button"
                        type="submit">Search</button>
            </div>
            <p class="govuk-body-s">
                Search using a record metadata term, for example – transferring body, series,
                consignment
                ref etc.
            </p>
        </form>
    </div>
</div>

    assert_contains_html(
        search_container_html,
        html,
        "p",
        {"class": "govuk-body search__heading"},
    )

```

Reply:
fixed with full search component html as one assert